src/libmongoc/tests/mock_server/mock-server.c

```diff
@@ -28,6 +28,7 @@
 #include "../test-conveniences.h"
 #include "../test-libmongoc.h"
 #include "../TestSuite.h"
+#include "../mongoc-test-settings.h"
```

Comment:
This may not be needed at all; the intent is to provide a central location for transporting settings across the client/mock-server/harness boundaries via global access (my real idea is a lightweight symbol table, not a bunch of free variables).